.github/workflows/ubuntu-builder.yml

```diff
@@ -1,20 +1,21 @@
-name: Ubuntu 22.04
-on:
-  workflow_dispatch:
-  push:
-    branches:
-      - main
-      - "release/*"
-  pull_request:
-    types: [assigned, opened, synchronize, reopened, labeled]
+name: Ubuntu builds
 
-env:
-  UBUNTU_2204_IMAGE: "ghcr.io/gofractally/psibase-builder-ubuntu-2204:7e3c6e5739df95ba33943789d2cdf5472f91111a"
+on:
+  workflow_call:
 
 jobs:
-  ubuntu-2204-build:
-    name: Ubuntu 22.04 | Build
+  build:
+    name: Ubuntu ${{ matrix.ubuntu-version }} - Build
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - builder-image: "ghcr.io/${{ github.repository_owner }}/psibase-builder-ubuntu-2004:b2a46a609b12f645aaeb03245afd0e3d2bf75340"
+            ubuntu-version: "2004"
+            docker-args: "-e LD_LIBRARY_PATH"
```

Comment:
`LD_LIBRARY_PATH` shouldn't be needed here, since it's already inside the docker image. If there is a definition of `LD_LIBRARY_PATH` from the host, we definitely don't want to pick it up.